cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -437,6 +434,9 @@ func main() {
 			parts := strings.Split(pprofHostPort, ":")
 			if len(parts) == 2 && parts[0] == "" {
 				pprofHostPort = fmt.Sprintf("localhost:%s", parts[1])
+			} else if parts[0] != "localhost" {
```

Comment:
Is this code suppose to be apart of this pr?

Reply:
I am creating a new PR.